docs/topics_5/rn-new-features.adoc

```diff
@@ -5,13 +5,62 @@
 
 This section describes the new features of the {ProductName} {DocInfoProductNumber}.
 
-Updated {WebName} interface::
+== Updated {WebName} interface
 The {WebName} has been redesigned with PatternFly 4. In addition to its general improved appearance, the {WebName} adds elements for project creation and management and updated transformation target icons.
 
-{ProductName} Operator for OpenShift Container Platform::
-The current release includes the {ProductName} Community Operator, which installs the {WebName} on OpenShift Container Platform 4.5 and later.
+== {ProductName} Operator for OpenShift Container Platform
+The current release includes version 2 of he {ProductName} Community Operator, which installs the {WebName} on OpenShift Container Platform 4.6 and later. Version 2 supports:
 
-Greater support for migrating from Spring Boot to Quarkus::
-The current release provides greater support for migrating from Spring Boot to Quarkus with the addition of 22 rules.
-+
-The new rules focus on the Spring Boot APIs that are implemented in Quarkus Spring extensions, where Spring Boot dependencies can be replaced by equivalent Quarkus extensions. Spring artifacts that do not have Quarkus equivalents are also reported.
+* HTTPS access
+* Category: Modernization & Migration
+* Horizontal scaling for Executor pod by {ProductName} instance YAML modification
+
+== Greater support for migrating from Spring Boot to Quarkus
+The release contains more rules to support migrating from Spring Boot to Quarkus.
+
+The new rules identify Spring Boot APIs that are implemented in Quarkus Spring extensions, where Spring Boot dependencies can be replaced by equivalent Quarkus extensions. Spring artifacts that do not have Quarkus equivalents are also reported.
+
+The rules identify the following technologies, which have equivalent Quarkus extensions:
+
+* Agroal
+* Amazon
+** Amazon DynamoDB
+** Amazon IAM
+** AWS Config
+** AWS Key Management Services (KMS)
+** AWS Lambda with HTTP APIs
+** AWS Lambda with AWS X-Ray
+** Amazon S3
+** Amazon Simple Email Service (SES)
```

Comment:
I would remove all the abbreviations if you are not going to use the abbreviations in the same section.